src/lib.rs

```diff
@@ -22,6 +22,14 @@
 //!     <th>Description
 //! <tr><td><code>alloc (default)</code>
 //!     <td>Enable features that require use of the heap, RSA in particular.
+//! <tr><td><code>less-safe-getrandom-custom</code>
+//!     <td>Treat a user-provided ("custom") <code>getrandom</code>
+//!         implementation as a secure random number generator (see
+//!         <code>SecureRandom</code>). Only has effect on targets
+//!         <strong>not</strong> supported by <code>getrandom</code>.
```

Comment:
This is misleading because `getrandom` does "support" these targets in some way and also the user will get to choose whether to use `rdrand` and/or a custom implementation depending on the target.